src/daemon/daemon.c

```diff
@@ -1360,16 +1360,17 @@ static gboolean notify_daemon_notify_handler(NotifyDaemonNotifications *object,
 	 */
 
 
-	if (g_variant_lookup(hints, "window-xid", "v", &data))
+	if (g_variant_lookup(hints, "window-xid", "@u", &data))
 	{
 		window_xid = (Window) g_variant_get_uint32 (data);
+		g_variant_unref(data);
 	}
 	/* deal with x, and y hints */
-	else if (g_variant_lookup(hints, "x", "v", &data))
+	else if (g_variant_lookup(hints, "x", "i", &x))
 	{
 		x = g_variant_get_int32 (data);
 
-		if (g_variant_lookup(hints, "y", "v", &data))
+		if (g_variant_lookup(hints, "y", "i", &y))
 		{
 			y = g_variant_get_int32 (data);
```

Comment:
Same as above -- the `g_variant_lookup` call assigns to `y`, so remove the `g_variant_get_int32` call.